Assignment3Test.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class Assignment3Test {
```

Comment:
Any tests for edge cases, like empty grid or grid with no words from dictionary?